src/NuGet.Core/NuGet.Protocol/Converters/V3SearchResultsConverter.cs

```diff
@@ -0,0 +1,106 @@
+using Newtonsoft.Json;
+using NuGet.Protocol.Model;
+using System;
+
+namespace NuGet.Protocol.Converters
+{
+    public class V3SearchResultsConverter : JsonConverter
```

Comment:
`NuGet.Protocol` is published as a NuGet package for anyone to use in their own applications. This means that any public class/method/property/field/etc becomes part of our public API surface, which we should avoid as much as possible. Customers should be using our abstractions, so they should not need to use our converter.

```suggestion
    internal class V3SearchResultsConverter : JsonConverter
```

Reply:
Ok. Fixed.